picturae_importer.py

```diff
@@ -76,11 +76,17 @@ def process_csv_files(self):
 
         # creating paths list for botany importer out of root directories in CSV file
 
```

Comment:
making sure that the paths list supplied to the botany class matches with the image_list, to prevent false runs.